content/en/security/application_security/enabling/serverless.md

```diff
@@ -216,6 +216,136 @@ To install and configure the Datadog Serverless Framework plugin:
 
 [3]: https://app.datadoghq.com/security/appsec?column=time&order=desc
 
+{{% /tab %}}
+{{% tab "Datadog CLI" %}}
+
+The Datadog CLI modifies existing Lambda function configurations to enable instrumentation without requiring a new deployment. It is the quickest way to get started with Datadog's serverless monitoring.
+
+**If you already have tracing enabled on your functions**, run the following command with the `--appsec` flag to enable threat detection, and skip steps 1-6 below.
+
+ ```sh
+  datadog-ci lambda instrument --appsec -f <functionname> -f <another_functionname> -r <aws_region> -v {{< latest-lambda-layer-version layer="python" >}} -e {{< latest-lambda-layer-version layer="extension" >}}
+```
+  - Replace `<functionname>` and `<another_functionname>` with your Lambda function names. </br>
+  - Alternatively, you can use `--functions-regex` to automatically instrument multiple functions whose names match the given regular expression.
+  - Replace `<aws_region>` with the AWS region name.
+
+  **Note**: Instrument your Lambda functions in a development or staging environment first. If the instrumentation result is unsatisfactory, run `uninstrument` with the same arguments to revert the changes.
+
+Additional parameters can be found in the [CLI documentation][2].
+
+**If you are configuring initial tracing for your functions**, perform the following steps:
+
+1. Install the Datadog CLI client:
+
+    ```sh
+    npm install -g @datadog/datadog-ci
+    ```
+
+2. If you are new to Datadog serverless tracing, launch the Datadog CLI in interactive mode to guide your first installation for a quick start, and ignore the remaining steps. To permanently install Datadog for your production applications, skip this step and follow the remaining steps to run the Datadog CLI command in your CI/CD pipelines _after_ your normal deployment.
```

Comment:
This works, but it's an added cognitive load for the reader cos you're asking them to make a choice inside the step and the command they need to run if they choose the first option is split by the second option. It's kind of a tough one cos this section is so dense already. Have you thought about breaking this into sub sections? For example:
### If this is your first time...  
steps 1 and 2

### If you've used x before and want to...
steps 1,3,4 etc

Reply:
Thanks @hestonhoffman that's great feedback and agree after re-reading this, it does get wordy and confusing, @sourabhkatti do you have some ideas how this should be re-structured a bit ?

Reply:
This makes sense, we can keep just the "new to datadog..." flow and remove the "existing to datadog..." flow. 

If a customer already knows the steps, they can intuitively skip ahead anyway